tests/Add-DbaAgDatabase.Tests.ps1

```diff
@@ -38,49 +47,130 @@ Describe "Add-DbaAgDatabase" -Tag "UnitTests" {
     }
 }
 
-Describe "Add-DbaAgDatabase" -Tag "IntegrationTests" {
+Describe "Add-DbaAgDatabase IntegrationTests" -Tag "IntegrationTests" {
```

Comment:
It's already tagged as integration, i liked the cleanliness of the previous Describe

Reply:
The tag is not included in the output. I'll make a screenshot next time I start my lab.

Reply:
This is the output when the tag name is also included in the name of the Describe:
<img width="697" height="169" alt="image" src="https://github.com/user-attachments/assets/54d2c0fa-3fdf-4e91-850e-14f3b9d6f129" />

This is the output of both Describe blocks have the same name:
<img width="691" height="142" alt="image" src="https://github.com/user-attachments/assets/145c056c-fb8d-458d-9292-2743234d5e04" />

I think this is not so important and I see your point that it is redundant in the code. And with the following names of the Context blocks I think it is clear what type of test are grouped together.